lightspeed/titles/lightspeed-release-notes/master.adoc

```diff
@@ -9,6 +9,6 @@ include::aap-common/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
 :context: lightspeed-release-notes
 
-== {LightspeedFullName} Release Notes
+= {LightspeedFullName} Release Notes
```

Comment:
Move the `include::aap-common/making-open-source-more-inclusive.adoc[leveloffset=+1]` line below the title